joy2twist/include/joy2twist/joy_input.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <stdexcept>
+#include <string>
+
+namespace joy2twist
+{
+
+class JoyInput
+{
+public:
+  enum class Type {
+    AXIS,
+    BUTTON,
+  };
+
+  Type type = Type::AXIS;
+  int index = -1;
+  bool is_inverted = false;
+
+  static constexpr char TYPE_AXIS = 'A';
+  static constexpr char TYPE_BUTTON = 'B';
+  static constexpr char INVERTED_PREFIX = '!';
```

Comment:
```suggestion
  static constexpr char kTypeAxis = 'A';
  static constexpr char kTypeButton = 'B';
  static constexpr char kInvertdPrefix = '!';
```
According to google standard we follow, constants should be named this way. I know this was not followed in `joy2twist_node.hpp`, but this is an old package before "standards" times. Can you please change it also there?
Also, constants should be defined before variables